dotcom-rendering/src/components/StickyBottomBanner.importable.tsx

```diff
@@ -209,6 +210,25 @@ const buildBrazeBanner = (
 	timeoutMillis: DEFAULT_BANNER_TIMEOUT_MILLIS,
 });
 
+const useCountryCode = (): CountryCode | undefined => {
+	const [localeCode, setLocaleCode] = useState<CountryCode | null>(null);
```

Comment:
We cannot change the API of `getLocalCode`, which returns a `null` if it fails.

I prefer turning the `null` into `undefined` at the very end of the custom hook, because it allows `setLocaleCode` and `getLocaleCode` to be consistent when methods have a similar name:

- `*LocaleCode` returns `CountryCode | null`
- `*CountryCode` returns `CountryCode | undefined`